src/NuGet.Protocol.Catalog/SimpleHttpClient.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    public class SimpleHttpClient : ISimpleHttpClient
+    {
+        private static readonly JsonSerializer JsonSerializer = CatalogJsonSerialization.Serializer;
```

Comment:
>JsonSerializer [](start = 47, length = 14)

If it wasn't for this `JsonSerializer`, `SimpleHttpClient` wouldn't be catalog specific. Instead of having a static property, could we instead accept a `JsonSerializer` in the constructor? What are the tradeoffs?

Reply:
I was favoring a simple client that should work on all V3 JSON. I think the name of the static class `CatalogJsonSerialization` is what is confusing.

Perhaps renaming it to `NuGetJsonSerialization` would clear this up?